package/MDAnalysis/auxiliary/XVG.py

```diff
@@ -64,17 +96,32 @@ def uncomment(lines):
         # if comment_position == 0, then the line is empty
 
 class XVGStep(base.AuxStep):
-    """ AuxStep class for .xvg file format 
+    """ AuxStep class for .xvg file format.
+
+    Extends the base AuxStep class to allow selection of time and 
+    data-of-interest fields (by column index) from the full set of data read 
+    each step.
 
     Parameters
     ----------
-    time_selector : int
+    time_selector : int | None, optional

```

Comment:
You are free ti choose where it should be documented. We do it in the init function.